packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -739,11 +740,16 @@ class DescopeWc extends BaseDescopeWc {
       // put the totp variable on the root element, which is the top level 'div' inside the shadowroot
       setTOTPVariable(
         this.shadowRoot.querySelector('div'),
-        screenState?.totp?.image
+        screenState?.totp?.image,
       );
 
       this.rootElement.replaceChildren(clone);
 
+      // we need to wait for all components to render before we can set its value
+      setTimeout(() =>
+        updateScreenFromScreenState(this.rootElement, screenState),
```

Comment:
So after applying this fix, we will see the former OTP code autofilled?

Reply:
Yep